src/backend/lang_c/c_cfg.rs

```diff
@@ -24,11 +25,9 @@ pub enum ActionNode {
     Call(String),
     Return,
     If,
-    While,
-    DoWhile,
```

Comment:
@kriw , any particular reason you removed While and DoWhile from CFGNode ? What is the equivalent output in CFG now? 

I am sorry if I missed any conversation related to this if it happened on Telegram.

Reply:
I think While/DoWhile is no longer needed since ctrl_flow_struct have While/DoWhile.
After CCFG is created, it will be transformed to structured CFG.